doc/src/sgml/libpq2.sgml

```diff
@@ -2555,12 +2569,12 @@ typedef struct
        or <literal>PGEVT_RESULTDESTROY</literal> events for this result or
        results copied from it.
 -->
-《マッチ度[57.208589]》<literal>PGEVT_RESULTCREATE</literal>イベントが受け取られた時、<parameter>evtInfo</parameter>ポインタは<structname>PGEventResultCreate *</structname>にキャストされなければなりません。
+<literal>PGEVT_RESULTCREATE</literal>イベントが受け取られた時、<parameter>evtInfo</parameter>ポインタは<structname>PGEventResultCreate *</structname>にキャストされなければなりません。
 <parameter>conn</parameter>は結果を生成するために使われた接続です。
 これは、結果と関連しなければならないすべての<literal>instanceData</literal>を初期化するために、理想的な場所です。
-イベントプロシージャが失敗すると、結果は消去され、失敗が伝播します。
+イベントプロシージャが失敗する（ゼロが返される）と、結果は消去され、失敗が伝播します。
 イベントプロシージャはそれ自身の結果オブジェクトを<xref linkend="libpq-PQclear"/>しようと試みてはいけません。
-失敗コードを返す時、<literal>PGEVT_RESULTDESTROY</literal>イベントは送られないのですべての消去が行われなくてはなりません。
+失敗コードを返す時、<literal>PGEVT_RESULTCOPY</literal>または<literal>PGEVT_RESULTDESTROY</literal>イベントは送られないのですべての消去が行われなくてはなりません。
```

Comment:
"that is, it will not receive <literal>PGEVT_RESULTCOPY</literal>  or <literal>PGEVT_RESULTDESTROY</literal> events for this result or   results copied from it."の訳が更新されていないようです。

Reply:
見逃してました。修正します。